src/AzureADB2C/Provider.php

```diff
@@ -219,9 +219,9 @@ public function logout($post_logout_uri)
     }
 
     /**
-     * @return array
+     * {@inheritdoc}
```

Comment:
@atymic cf. https://github.com/SocialiteProviders/Providers/pull/1284#discussion_r1821659496
Maybe we can remove the docblocks too, let me know

Reply:
Yep may as well if you're changing it anyway

Reply:
I've removed them; see 09a0576908210bd9fad5eb7f762555cf88f6293d